Development/Building/FreeBSD.md

```diff
@@ -1,57 +1,254 @@
 # Building on FreeBSD
 
-Building on FreeBSD, much like Linux support, is made possible with [cctools-port](https://github.com/tpoechtrager/cctools-port). To setup Procursus on FreeBSD
+## System preparation
 
-## Install dependencies
-
-You can install dependencies with ``pkg``
+Mount `/proc`
 
+```sh
+sudo mount -t linprocfs none /proc
 ```
-sudo pkg autoconf automake bash cmake coreutils docbook-xsl dpkg fakeroot findutils gettext git gmake gnugrep gnupg gsed gtar libtool ncurses openssl patch perl5 pkgconf po4a python39 wget zstd
+
+Add mount entry to `/etc/fstab`
+
+```sh
+sudo sh -c 'echo "none /proc linprocfs rw" >> /etc/fstab
 ```
 
-You'll also need to install ``triehash``
+## Install dependencies
+
+Install dependencies with `pkg`.
 
+```sh
+sudo pkg install autoconf automake bash cmake coreutils docbook-xsl dpkg fakeroot findutils gettext git gmake gnugrep gnupg gsed gtar libtool libplist ncurses openssl patch perl5 pkgconf po4a python39 wget zstd
 ```
+
+You'll also need to install `triehash`
+
+```sh
 wget -O triehash https://raw.githubusercontent.com/julian-klode/triehash/main/triehash.pl
-gsed -i 's@#!/usr/bin/perl -w@#!/usr/bin/env perl -w@g' triehash
-sudo mv triehash /usr/local/bin
+gsed -i 's|#!/usr/bin/perl -w|#!/usr/bin/env perl -w|g' triehash
+sudo install -m755 triehash /usr/local/bin
 ```
 
-## Installing the toolchain
+## Install toolchain
+
+### Automatic install
 
-[This script](https://gist.github.com/asdfugil/71cdfca5aa1bc0d59de06518cd1c530c) will setup your SDKs, cctools-port with your iOS toolchain, and other dependencies needed.
+[This script](https://gist.github.com/asdfugil/8d629aa1d9f5080a4c6c868b47600879) will setup your SDKs, cctools-port with your iOS toolchain, and other dependencies needed.
 
 To run the script
 
+```sh
+wget -qO- https://gist.githubusercontent.com/asdfugil/8d629aa1d9f5080a4c6c868b47600879/raw/d845a136b5e1d447d8bd5a042199f8a309d684ab/procursus-toolchain-fbsd.sh | bash
 ```
-bash procursus-utils-fbsd.sh
+
+Then, add the following to your shell's startup file (such as ~/.profile)
+
+```sh
+export PATH=${HOME}/.local/bin:${PATH}
+export GNUBINDIR=${HOME}/.local/libexec/gnubin
 ```
 
-It's recommended that you add this script to your shells' config file, so that it's sourced on every environment on login.
+### Manual install
+
+If you want to install the toolchain manually for whatever reason,
+
+Export the location where you want to install the toolchain.
```

Comment:
```suggestion
If you want to install the toolchain manually for whatever reason, begin with exporting the location where you want to install the toolchain.
```

Try making this more seamless than a cliffhanging sentence. The suggestion clips the two together. Other than this, it lgtm.